docs/getting-started/gcp.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: docs-getting-started
+title: Running on Google Cloud Platform (GCP)
+slug: gcp
+toc: toc-user-guide.html
+redirect_from:
+  - /docs/platforms/gcp
+---
+
+This guide takes you through the steps to get Node-RED running on an GCP
+Virtual Machine instance.
+
+#### Create the base image
+
+1. Log in to the [Google Cloud Platform Console](https://console.cloud.google.com/)
+
+1. Select your project.
+
+1. Click `VPC network` >> `VPC networks` >> `Firewall rules` >> `CREATE FIREWALL RULE`
+
+   new `Create a firewall rule` with the options set as:
+     - Name: node-red-editor
+     - Network: default
+     - Priority: 1010
+     - Direction of traffic: Ingress
+     - Action on match: Allow
+     - Targets: ALL Instances in the network
+     - Source filter: IP ranges
+     - Source IP ranges: 0.0.0.0/0 
+     - Protocols and ports: Specified protocols and ports
+       - tcp: 1880
+
+   Click `Create` on the Settings page.
+
+1. Click `Compute Engine` >> `VM instances` >> `Create`
+
+   Give your machine info 
+     - Name: node-red-instance
+     - Region: us-central1
+     - Zone: us-central1-a
+     - Machine configuration
+       - Machine family
+         - General-purpose
+       - Series
+         - N1
+       - Machine type
+         - f1-micro
+     - Boot disk
+       - Ubuntu 18.04 LTS
+
+   Click `Create` on the Settings page, then the instance will start to be created.
+ 
+After a couple of minutes your instance will be running. In the console
+you can find your instance's IP address as `External IP`.
+
+#### Setup Node-RED
+
+The next task is to log into the instance then install node.js and Node-RED.
+
+Log into your instance using the authentication details you specified in the
+previous stage.
+
+Once logged in you need to install node.js and Node-RED
+
+       curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
```

Comment:
Does the default Linux install script not work for some reason ?

Reply:
Thank you for checking.
Check it yourself!

Reply:
@dceejay 

Hi

I corrected manual install document referring to the official documentation of Node.js --> [NodeSource Node.js Binary Distributions](https://github.com/nodesource/distributions#manual-installation)

+ modified

https://github.com/node-red/node-red.github.io/pull/139/files#diff-4f1b2783aabc91314b49b74b359d7f7aR56-R94

I want to make sure that it is feasible with the documentation.